packages/blocks/src/api/constants.js

```diff
@@ -0,0 +1,7 @@
+export const DEPRECATED_ENTRY_KEYS = [
```

Comment:
We should include a JSDoc describing the purpose of this value.

Reply:
On it!